draft-ietf-oauth-transaction-tokens.md

```diff
@@ -433,6 +443,13 @@ grant_type=urn%3Aietf%3Aparams%3Aoauth%3Agrant-type%3Atoken-exchange
 ~~~
 {: #figtxtokenrequest title="Example: Txn-Token Request"}
 
+### Subject Token Types {#subject-token-types}
+The `subject_token_type` parameter value MUST be a URI {{RFC3986}}. It MAY be any one of the subject token types described in Section 3 of OAuth 2.0 Token Exchange {{RFC8693}} except the Refresh Token type (i.e., `urn:ietf:params:oauth:token-type:refresh_token`), or it MAY be set to the value:
+
+`urn:ietf:params:oauth:token-type:self_signed`
```

Comment:
Is defining and registering a new token type actually necessary? There is urn:ietf:params:oauth:token-type:jwt

Reply:
This will certainly simplify the current PR, but would it prevent us in future from using the `urn:ietf:params:oauth:token-type:jwt` subject token type for other unanticipated purposes in the request?

Reply:
Good point. I also learned that token-type:jwt more or less defines the format of the token and not the purpose. Than again the name "token-type:self_signed" also seems to describe format rather than purpose.

How about registering a token type name based on the purpose and maybe avoid having "signed" in the type name while still defining the format as JWT with the needed attributes for interop and allow implementers to choose whether they want to require a signed JWT or allow a plain one. If implementers require a signed one, the additional attributes like "exp" could make sense to help prevent misuse/abuse. We could make those additional attributes OPTIONAL in the plain version and REQUIRED in the signed one.